index.html

```diff
@@ -154,6 +154,39 @@ <h3 class="faq__question">Can I make changes if the product is not what I expect
             </div>
         </div>
     </section>
+    <footer class="footer">
+        <div class="footer__container-ancor">
+            <div>
+                <img class="footer__img" src="./images/icon.jpg" alt="icono">
+                <div class="footer__social-media">
+                    <a href="https://www.facebook.com/nicolle.montillaruiz/"><i class="fab fa-facebook-f"></i></a>
+                    <a href="https://www.instagram.com/"><i class="fab fa-instagram"></i></a>
+                    <a href="https://twitter.com/home"><i class="fab fa-twitter"></i></a>
+                </div>
+            </div>
+            <div class="footer__links">
+                <a href="#">Home</a>
+                <a href="#">Contact</a>
+                <a href="#">Offices</a>
+                <a href="#">Stores</a>
+                <a href="#">About</a>
+            </div>
+            <div class="footer__links">
+                <a href="#">Home</a>
+                <a href="#">Contact</a>
+                <a href="#">Offices</a>
+                <a href="#">Stores</a>
+                <a href="#">About</a>
+            </div>
+        </div>
+        <div class="footer__politics">
+            <p class="footer__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit quae tempore, assumenda nisi aut quidem suscipit ipsa atque necessitatibus libero.</p>
+            <div class="footer__terms">
+                <a class="footer__anc-term footer__anc-term--ancor" href="#">Terms of Service | Privacy Policy</a>
```

Comment:
change `footer__anc-term footer__anc-term--ancor` to `footer__terms-links footer__terms-links--no-underline`